src/main/java/service/UserService.java

```diff
@@ -0,0 +1,23 @@
+package service;
+
+import model.User;
+import repositoriy.AllUsersRepository;
+import java.util.List;
+import java.util.Optional;
+
+public class UserService {
+
+    public static UserService userService = new UserService();
+    public AllUsersRepository allUsersRepository = AllUsersRepository.getInstance();
+
+    public UserService() {
+    }
+
+    public UserService(AllUsersRepository allUsersRepository) {
```

Comment:
This constructor is redundant, repository is singleton and can be accessed everywhere in the code by _getInstance()_ method